tests/test_unit/test_transform_utils.py

```diff
@@ -0,0 +1,50 @@
+import dask.array as da
+import numpy as np
+import pytest
+from skimage import transform
+
+from brainglobe_template_builder.preproc.transform_utils import (
+    downsample_anisotropic_image_stack,
+)
+
+
+@pytest.fixture()
+def stack():
+    """Create a dask array representing an image stack"""
+    data = np.random.rand(10, 100, 100)  # Random image stack
+    return da.from_array(data, chunks=(1, 100, 100))
+
+
+@pytest.fixture()
+def not_slicewise_stack():
+    """Create a dask array representing an image stack"""
+    data = np.random.rand(10, 100, 100)  # Random image stack
+    return da.from_array(data, chunks=(2, 100, 100))
+
+
+def test_downsample_anisotropic_image_stack(stack):
+    """Test that downsampling with dask gives same as without."""
+    xy_downsampling = 20
+    z_downsampling = 2
```

Comment:
What happens if the input image shape cannot be integer-divided into the downsampling factors.
Do we want to test for that case (e.g. via parametrising across factors)?
Should the user (we) take care with choosing "nice" factors?

Reply:
See below and new docstring. I suggest we test when this causes us a bug, in the interest of progress. The test would be slightly more complex, because we need to calculate analytically the expected solution on the borders.